easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2006-intel-2020a.eb

```
@@ -0,0 +1,41 @@
+name = 'OpenFOAM'
+version = 'v2006'
+
+homepage = 'https://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://sourceforge.net/projects/openfoam/files/%(version)s/']
+sources = [SOURCE_TGZ]
+patches = [
+    'OpenFOAM-%(version)s-cleanup.patch',
+    'OpenFOAM-v1906-wmake-ompi.patch',
+]
+checksums = [
+    '30c6376d6f403985fc2ab381d364522d1420dd58a42cb270d2ad86f8af227edc',  # OpenFOAM-v2006.tgz
+    'cb3c6d26569cf11d22941963acb9741238fe829f21b452506a51a71b5c42ae84',  # OpenFOAM-v2006-cleanup.patch
+    '518e27683c5c41400cfbc17b31effa50b31b25916dccbf85b18b0b955f642505',  # OpenFOAM-v1906-wmake-ompi.patch
+]
+
+dependencies = [
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    # OpenFOAM requires 64 bit METIS using 32 bit indexes (array indexes)
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '6.0.9'),
+    ('CGAL', '4.14.3', '-Python-3.8.2'),
+    ('ParaView', '5.8.0', '-Python-3.8.2-mpi'),
```

@ekieffer `gnuplot` dependency is missing (see #11568)

```suggestion
    ('ParaView', '5.8.0', '-Python-3.8.2-mpi'),
    ('gnuplot', '5.2.8'),
```